### ISSUE TITLE
Go quick start issue

### ISSUE THREAD

**Issue by ronaldpetty** [grpc/grpc.io#177]
Go quick start (https://grpc.io/docs/quickstart/go/) appears to not work based on instructions.  I was unable to regenerate helloworld.pb.go as protoc would not regenerate the gRPC server component.  Downgrading seems to fix it (there is upstream work that caused this).

**Steps to reproduce the issue**
1. Follow steps in guide up to adding a new server rpc
2. run protoc fails

```
 helloworld % protoc -I helloworld helloworld/helloworld.proto --go_out=plugins=grpc:helloworld
2020/04/04 16:42:33 WARNING: Missing 'go_package' option in "helloworld.proto", please specify:
	option go_package = ".;helloworld";
A future release of protoc-gen-go will require this be specified.
See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.

--go_out: protoc-gen-go: plugins are not supported; use 'protoc --go-grpc_out=...' to generate gRPC
 helloworld %
```
**Expected behavior**
Adding rpc to server would work.

**Additional info**
I was able to get it working after reading the following:

https://stackoverflow.com/questions/60578892/protoc-gen-go-grpc-program-not-found-or-is-not-executable

and applying this:

```
GIT_TAG="v1.2.0" # change as needed
go get -d -u github.com/golang/protobuf/protoc-gen-go
git -C "$(go env GOPATH)"/src/github.com/golang/protobuf checkout $GIT_TAG
go install github.com/golang/protobuf/protoc-gen-go
```

from https://github.com/golang/protobuf.  Also added `syntax = "proto3";` and removed `option java_package = "io.grpc.examples.helloworld"` in the proto.

I am probably missing an already existing ticket, but was unable to find one.  While I got it working, I don't know if my hack is 'good' enough or even the correct way.

**Comment by chalin** [comment]
I've followed the instructions in the Go Quick Start and I'm unable to reproduce the problem that you are reporting. Can you provide more information such as OS, version of Go, etc.

**Comment by dapengzhang0** [comment]
cc @easwars

**Comment by easwars** [comment]
cc @menghanl

**Comment by menghanl** [comment]
My guess is the `protoc-gen-go` in PATH is not the correct version.

There are two of them, from different modules: https://github.com/golang/protobuf/releases#v1.4-generated-code
Please make sure you have the one with gRPC support installed, and try again.

**Comment by chalin** [comment]
Thanks for following up @menghanl.

@ronaldpetty - I'm going to close this issue for now. If the problem persists, feel free to reopen the issue for file a new one. (Note that the Go Quick Start updates have landed - #183).

**Comment by ronaldpetty** [comment]
Thats everyone, I went back and tried to recreate my disaster-piece.  I believe issue was trying to figure out https://blog.golang.org/protobuf-apiv2 and then went back to the quick start because I was having some troubles.

I was able to just 'start' with the quick start and it work.  Sorry for the troubles.
